addon/utils/_private/rdfa-utils.ts

```diff
@@ -315,3 +326,90 @@ export function isLinkToNode(triple: OutgoingTriple): triple is LinkTriple {
     triple.object.termType === 'ResourceNode'
   );
 }
+
+export type AddPropertyArgs = {
+  /** The resource to which to add a property */
+  resource: string;
+  /** Property to add */
+  property: OutgoingTriple;
+  /**
+    A transaction to use in place of getting a new one from state.tr
```

Comment:
This comment refers to a transaction but it's on a member of type `EditorState`. In fact, I can't see this type being used anywhere?